### PR TITLE
WEBUI-928: analysis of import create button disabled feature

### PR DESCRIPTION
**Pull request by Nishant0928**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,7 +115,8 @@
               ],
               blob: ['appLinks', 'preview'],
             },
-            // expressions: { eval: false },
+
```
Comment (swarnadipa-dev) [inline index.html]: Please remove extra lines and duplicate code

File: elements/document/nuxeo-document-import.js [hunk 2]
```diff
@@ -40,9 +40,12 @@ import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-tooltip.js';
 import '@nuxeo/nuxeo-ui-elements/nuxeo-document-layout.js';
 import '../nuxeo-document-creation-stats/nuxeo-document-creation-stats.js';
 import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+import { config } from '@nuxeo/nuxeo-elements';
 import { html } from '@polymer/polymer/lib/utils/html-tag.js';
 import { DocumentCreationBehavior } from '../nuxeo-document-creation/nuxeo-document-creation-behavior.js';
 
+const flag = config.get('createDisabled.flag', false);
```
Comment (swarnadipa-dev) [inline elements/document/nuxeo-document-import.js]: Please use a more meaningful name for the flag so that its easy to understand what it is doing.

**Review by swarnadipa-dev — CHANGES_REQUESTED** [review]
Please change the commit message to remove the word 'analysis', as this is not an analysis, but an implementation

**Review by swarnadipa-dev — DISMISSED** [review]
Looks good

**Review by poonamyadav252 — COMMENTED** [review]
It is fine for me and don't forget to connect with Nisha to review the functionality.

**Review by rajatgoel-gl — CHANGES_REQUESTED** [review]
There are so many formatting changes have been introduced in the checking file. Kindly commit only changes that are required for the WEBUI-928

**Comment by Nishant0928** [comment]
> There are so many formatting changes have been introduced in the checking file. Kindly commit only changes that are required for the [WEBUI-928](https://jira.nuxeo.com/browse/WEBUI-928)

@rajatgoel-gl , the formatting changes have occured due to the prettier.

**Review by poonamyadav252 — CHANGES_REQUESTED** [review]
Kindly rebase your branch with maintenance.

**Pull request closed without merging**